RA-Policy.md

```diff
@@ -1,47 +1,21 @@
-* Ensure the security categorization decision is reviewed and approved by the Authorizing Official (AO) or authorizing official designated representative.
-
-### Risk Assessment
-* 18F must conduct an assessment of risk, including the likelihood and magnitude of harm, from the unauthorized access, use, disclosure, disruption, modification, or destruction of the information system and the information it processes, stores, or transmits
-* Document risk assessment results in vulnerability scan reports, security assessment reports, plan of actions and milestones
-* Reviews risk assessment results are conducted on a as needed basis
-* Updates the risk assessment annually or whenever there are significant changes to the information system or environment of operation (including the identification of new threats and vulnerabilities), or other conditions that may impact the security state of the system.
-
-### Vulnerability Scanning
-* 18F must conduct scans for vulnerabilities within information systems and hosted applications on a weekly basis and on-going for continuous monitoring and incident response processes when new vulnerabilities potentially affecting the system/applications are identified and reported
-* Must employ vulnerability scanning tools and techniques that promote interoperability among tools and automate parts of the vulnerability management process by using standards for:
- * Enumerating platforms, software flaws, and improper configurations;
- * Formatting and making transparent, checklists and test procedures; and
- * Measuring vulnerability impact;
-* Analyzes vulnerability scan reports and results from security control assessments
-* Conduct remediation on legitimate vulnerabilities (Highs within 30 days, Moderates within 90 days, Lows within 120 days) in accordance with an organizational assessment of risk
-* Shares information obtained from the vulnerability scanning process and security control assessments with designated personnel throughout the organization to help eliminate similar vulnerabilities in other information systems (i.e., systemic weaknesses or deficiencies).
-
-## 4. Roles and Responsibilities
-|Roles                  |Responsibilities|
-|-----------------------|------------------------------------------------------------------------------------------------------------|
-|Security Operations    | Perform all risk assessment tasks to meet compliance standards and continuous monitoring of 18F information systems and enterprise networks|
-|Senior Director        | Ensuring the Policy is approved, implemented and communicated.|
-|Director of Compliance | Owner of the Policy. Ensuring the Policy meets the compliance requirements.|
+# Risk assessment policy
+
+See [CIO P 2100.1J – GSA IT Security Policy](http://www.gsa.gov/portal/mediaId/129634/fileName/CIO_21001J_CHGE_1_GSA_Information_Technology_(IT)_Security_Policy_(Posted_Version_4-28-2016).action) Chapter 5, _Policy on Technical Controls_, which covers:
```

Comment:
Hello,

I'm thinking you accidentally added me to this? I'm not sure what this is
for...



On Tue, Sep 13, 2016 at 12:23 PM, Chris Nelson <notifications@github.com>
wrote:

> In RA-Policy.md
> <https://github.com/18F/compliance-docs/pull/9#discussion_r78594166>:
>
> > - * Enumerating platforms, software flaws, and improper configurations;
> > - * Formatting and making transparent, checklists and test procedures; and
> > - * Measuring vulnerability impact;
> > -* Analyzes vulnerability scan reports and results from security control assessments
> > -* Conduct remediation on legitimate vulnerabilities (Highs within 30 days, Moderates within 90 days, Lows within 120 days) in accordance with an organizational assessment of risk
> > -* Shares information obtained from the vulnerability scanning process and security control assessments with designated personnel throughout the organization to help eliminate similar vulnerabilities in other information systems (i.e., systemic weaknesses or deficiencies).
> > -
> > -## 4. Roles and Responsibilities
> > -|Roles                  |Responsibilities|
> > -|-----------------------|------------------------------------------------------------------------------------------------------------|
> > -|Security Operations    | Perform all risk assessment tasks to meet compliance standards and continuous monitoring of 18F information systems and enterprise networks|
> > -|Senior Director        | Ensuring the Policy is approved, implemented and communicated.|
> > -|Director of Compliance | Owner of the Policy. Ensuring the Policy meets the compliance requirements.|
> > +# Risk assessment policy
> > +
> > +See [CIO P 2100.1J – GSA IT Security Policy](http://www.gsa.gov/portal/mediaId/129634/fileName/CIO_21001J_CHGE_1_GSA_Information_Technology_(IT)_Security_Policy_(Posted_Version_4-28-2016).action) Chapter 5, _Policy on Technical Controls_, which covers:
>
> @NoahKunin <https://github.com/NoahKunin> I don't see RA controls listed
> in chapter 5 of this document. I see them in chapter 3. Do we need to tweak
> this to point to the correct place in the GSA IT Security Policy or am I
> missing something?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/18F/compliance-docs/pull/9/files/86738e076637a2ee41374a36be50a3002c3b78cc#r78594166>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ARrehAMTlyBwuAnzvPXB1IDZRHA-_EUqks5qps3pgaJpZM4J7JXW>
> .
>



-- 
Whitney Wissinger
Events & Facilities
(c) 703.980.6118
Center for Open Science
https://cos.io/